src/partial-templates/partial-templates.controller.ts

```diff
@@ -1,9 +1,11 @@
-import { Controller, Get, Post, Body, Patch, Param, Delete, ParseIntPipe, UseGuards } from '@nestjs/common';
+import { Controller, Get, Post, Body, Patch, Param, Delete, ParseIntPipe, UseGuards, Query, Res, HttpStatus, UseFilters } from '@nestjs/common';
 import { PartialTemplatesService } from './partial-templates.service';
 import { CreatePartialTemplateDto } from 'src/models/partialTemplate/create-partial-template.dto';
 import { UpdatePartialTemplateDto } from 'src/models/partialTemplate/update-partial-template.dto';
-import { JwtAuthGuard } from 'src/auth/strategies/guards/jwt-auth.guard';
+import { Response } from 'express';
+import { CustomExceptionFilter } from 'src/common/filters/custom-exception.filter';
 
+@UseFilters(CustomExceptionFilter)
```

Comment:
Delete import and `@UseFilters` because `CustomExceptionFilter` is enabled at app level, check `main.ts`

```js
  app.useGlobalFilters(new CustomExceptionFilter());
```
_main.ts_